ember/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mysagw",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
Please also bump both versions in the main compose file. The one for the ember image and the one at the top of the file.

Reply:
Done